pkg/addons/data/Secret/default/registry-config

```diff
@@ -1,8 +1,37 @@
 apiVersion: v1
-data:
-  config.yml: YXV0aDoKICBvcGVuc2hpZnQ6CiAgICByZWFsbTogb3BlbnNoaWZ0Cmh0dHA6CiAgYWRkcjogOjUwMDAKbG9nOgogIGxldmVsOiBkZWJ1ZwptaWRkbGV3YXJlOgogIHJlZ2lzdHJ5OgogIC0gbmFtZTogb3BlbnNoaWZ0CiAgcmVwb3NpdG9yeToKICAtIG5hbWU6IG9wZW5zaGlmdAogICAgb3B0aW9uczoKICAgICAgYWNjZXB0c2NoZW1hMjogdHJ1ZQogICAgICBlbmZvcmNlcXVvdGE6IGZhbHNlCiAgICAgIHB1bGx0aHJvdWdoOiB0cnVlCiAgc3RvcmFnZToKICAtIG5hbWU6IG9wZW5zaGlmdApzdG9yYWdlOgogIGF6dXJlOgogICAgYWNjb3VudGtleTogJyoqKiBHRU5FUkFURUQgKioqJwogICAgYWNjb3VudG5hbWU6ICcqKiogR0VORVJBVEVEICoqKicKICAgIGNvbnRhaW5lcjogcmVnaXN0cnkKICAgIHJlYWxtOiBjb3JlLndpbmRvd3MubmV0CiAgY2FjaGU6CiAgICBibG9iZGVzY3JpcHRvcjogaW5tZW1vcnkKICBkZWxldGU6CiAgICBlbmFibGVkOiB0cnVlCnZlcnNpb246IDAuMQo=
 kind: Secret
 metadata:
   name: registry-config
   namespace: default
 type: Opaque
+stringData:
```

Comment:
moving data->stringData and adding the REGISTRY_STORAGE_ACCOUNT_KEY text is entirely pragmatic for now but implies a little technical debt around automatically syncing in updated yaml objects from a cluster.  Not a blocker, but just keep it in mind :)